homework_3/solution.js

```diff
@@ -0,0 +1,56 @@
+// solution.js (called by test.js)
+// module.exports = function (equation) {
+//      return '<Your result here>';
+//  }
+
+// function equation (Result) {   
+ 
+    var colors = require('colors');
+    var Result;
+    var NextOperand;
+    var Operator;
+    var IndexStart;
+
+    IndexStart = 4; // assume min 2 operands and go directly to 4th arg to start calc
+
+    Result = Number(process.argv[2]);
+    for (var i=IndexStart; i < process.argv.length; i+=2) {
+        NextOperand = Number(process.argv[i]);
+        Operator = (process.argv[i-1]);
+        if (Operator === '+') {
```

Comment:
Might be cleaner to use a `switch` here. I believe `switch` is also more performant than if/elses.